### DISCUSSION TITLE
Tsynth in White

### DISCUSSION THREAD

**Discussion by throx1971** [ElectroTechnique/electrotechnique.github.io#70] [Hardware Issues]
![Tsynth White 1](https://user-images.githubusercontent.com/89705663/131227997-32b8f9ca-9fa6-4580-a13f-ca73e5b71fd2.jpg)
![Tsynth White 2](https://user-images.githubusercontent.com/89705663/131228003-1958db48-60f3-47ac-a362-e75a74e954df.jpg)

With pink Leds

**Comment by Electrotechnique** [comment]
Oberheim!

![image](https://user-images.githubusercontent.com/50934180/131325680-fda9298b-d728-432c-9b39-6d6483b98eb1.png)
